stable/datadog/README.md

```diff
@@ -255,6 +255,7 @@ helm install --name <RELEASE_NAME> \
 | `rbac.serviceAccountName`                | existing ServiceAccount to use (ignored if rbac.create=true)                              | `default`                                   |
 | `daemonset.podLabels`                    | labels to add to each pod                                                                 | `nil`                                       |
 | `datadog.name`                           | Container name if Daemonset or Deployment                                                 | `datadog`                                   |
+| `datadog.dogStatsDPort`                  | Used to override the default agent DogStatsD Port                                         | `8125`                                      |
```

Comment:
could you move this parameter near others dogstatsd parameters: https://github.com/helm/charts/pull/18966/files#diff-d1853bc077f8540ce703988dee21395dL274

Reply:
@clamoriniere  done :)